src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -181,6 +191,23 @@ public ServiceMetadata GetServiceMetaData(string org, string service)
             }
             catch (Exception ex)
             {
+                if (ex is DirectoryNotFoundException || ex is FileNotFoundException)
```

Comment:
This code is outdated, this part was removed in pr a couple of days ago and should not be readded unless you have a good reason for doing so

Reply:
changed it to match what's in master